src/components/tx/SponsoredBy/index.tsx

```diff
@@ -11,43 +11,72 @@ export const SPONSOR_LOGOS = {
   [chains.gor]: '/images/common/token-placeholder.svg',
 }
 
-const SponsoredBy = ({ relays, tooltip }: { relays: RelayResponse; tooltip?: string }) => {
+const SponsoredBy = ({
+  relays,
+  tooltip,
+  shouldRelay,
+}: {
+  relays?: RelayResponse
+  tooltip?: string
+  shouldRelay: boolean
+}) => {
   const chain = useCurrentChain()
 
   return (
     <Box className={css.sponsoredBy}>
       <SvgIcon component={GasStationIcon} inheritViewBox className={css.icon} />
-      <div>
-        <Stack direction="row" spacing={0.5} alignItems="center">
-          <Typography variant="body2" fontWeight={700} letterSpacing="0.1px">
-            Sponsored by
+
+      {shouldRelay ? (
+        <div>
+          <Stack direction="row" spacing={0.5} alignItems="center">
+            <Typography variant="body2" fontWeight={700} letterSpacing="0.1px">
+              Sponsored by
+            </Typography>
+
+            <img src={SPONSOR_LOGOS[chain?.chainId || '']} alt={chain?.chainName} className={css.logo} />
+            <Typography variant="body2" fontWeight={700} letterSpacing="0.1px">
+              {chain?.chainName}
+            </Typography>
+
+            {tooltip ? (
+              <Tooltip title={tooltip} placement="top" arrow>
+                <span style={{ display: 'flex' }}>
+                  <SvgIcon
+                    component={InfoIcon}
+                    inheritViewBox
+                    color="info"
+                    fontSize="small"
+                    sx={{ verticalAlign: 'middle', color: '#B2B5B2' }}
+                  />
+                </span>
+              </Tooltip>
+            ) : null}
+          </Stack>
+
+          <Typography variant="body2" color="primary.light">
+            Transactions per hour:{' '}
+            <Box component="span" sx={{ fontWeight: '700', color: 'text.primary' }}>
+              {relays?.remaining ?? 0} of {relays?.limit ?? 0}
+            </Box>
+            {relays && !relays.remaining && (
+              <Box component="span" sx={{ color: 'error.main' }}>
+                {' '}
+                &mdash; will reset in the next hour
+              </Box>
```

Comment:
I think this is slightly redundant since the label already says "Transactions per hour". Maybe we could instead make the "0 of 5" a red color?

Reply:
I added this based on Liliya's feedback. It might be redundant, but it's definitely clearer. IMO we should err on the safe side.